.github/workflows/ec2-integration-test.yml

```diff
@@ -100,14 +107,15 @@ jobs:
               -var="arc=${{ matrix.arrays.arc }}" \
               -var="binary_name=${{ matrix.arrays.binaryName }}" \
               -var="local_stack_host_name=${{ inputs.localstack_host }}" \
-              -var="s3_bucket=${{env.S3_INTEGRATION_BUCKET}}" \
+              -var="region=${{ inputs.region }}" \
+              -var="s3_bucket=${{ inputs.s3_integration_bucket }}" \
               -var="plugin_tests='${{ github.event.inputs.plugins }}'" \
               -var="excluded_tests='${{ matrix.arrays.excludedTests }}'" \
               -var="ssh_key_name=${{env.KEY_NAME}}" \
               -var="test_dir=${{ matrix.arrays.test_dir }}" \
-              -var="agent_start=${{ matrix.arrays.agentStartCommand }}"; then terraform destroy -auto-approve
+              -var="agent_start=${{ matrix.arrays.agentStartCommand }}"; then terraform destroy -var="region=${{ inputs.region }}" -var="ami=${{ matrix.arrays.ami }}" -auto-approve
```

Comment:
Now destroy is not allowed to use default values and for some reason the terraform is not saving the pass in values. So to get destroy to work we need pass in these fields.